src/components/profilePopup.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import profileImagePlaceHolder from '../assets/images/profile_plaholder.png';
+
+const ProfileView = ({
+  image, firstName, lastName, dob, bio, username, removeModel, follow, unfollow, followThem,
+}) => (
+    <>
+    <div id="myModal" className="modal">
+        <div className="modal-content">
+        <span id="close" className="close" onClick={removeModel}>&times;</span>
+        <div className="profile">
+        <div className="profile__imagecontent">
+            <img src={image || profileImagePlaceHolder} className="profile__imagecontent"/>
+        </div>
+        <div className="profile__profilecontent">
+            <h6 className="profile__profilecontent_h">Username:</h6> <p className="profile__profilecontent_p">{username}</p>
+            <h6 className="profile__profilecontent_h">First Name:</h6> <p className="profile__profilecontent_p">{firstName}</p>
+            <h6 className="profile__profilecontent_h">Last Name:</h6> <p className="profile__profilecontent_p">{lastName}</p>
+            <h6 className="profile__profilecontent_h">Date of birth:</h6> <p className="profile__profilecontent_p">{dob}</p>
```

Comment:
we do not have date of birth in our mockups please remove.

Reply:
But we can still have it because in our database we have it and it can be included on a given user profile.